docs/conf.py

```diff
@@ -70,13 +70,11 @@
     'reno.sphinxext',
     'nbsphinx',
     'sphinx_design',
-    "sphinxcontrib.jquery",  # Remove when changing html_theme to qiskit_ecosystem.
+    "qiskit_sphinx_theme",
```

Comment:
`qiskit_sphinx_theme` already activates `sphinxcontrib.jquery`. This is more future proof for the upcoming Sphinx Theme 1.12, which will provide `versionutils.py` for you.